snuba/clickhouse/columns.py

```diff
@@ -218,7 +221,7 @@ def for_schema(self) -> str:
         return "Nullable({})".format(self.inner_type.for_schema())
 
     def get_raw(self) -> ColumnType:
-        return self.inner_type.get_raw()
+        return Nullable(self.inner_type.get_raw())
```

Comment:
I am not sure I fully understand how we are going to use this going on.
From the discussion on the previous PR we were removing all the types from the schema that are only used in DDL and that do not change how the table schema is viewed at runtime (like materialized and WithDefault). 

Now `Nullable` basically becomes a raw type. Does this mean that when we need to look for the type of the column (like in a signature validator) we will manually unnpack Nullable? Is this going to be the case inn other types ?

Reply:
Yes, it means Nullable is part of the raw type and signature validator (the only place we look for this) is manually unpacking Nullable. The only other scenario like that is perhaps Array/Nested, which is also part of the raw type, though I'm not sure to what extent this is supported in the validator.